### ISSUE TITLE
Not able to find these functions

### ISSUE THREAD

**Issue by pqrsh** [robot-learning-freiburg/MM-DistillNet#26]
Hi, 
In the following code snippet I'm not able to find the functions **train_adversarial, train_adversarial_multiteacher, train_adversarial_multiteacher_wgangp**. Can you please help me out in finding these functions?

loss = train_traditional(
                train_set,
                model,
                optimizer,
                epoch,
                config,
                writer
            )
        elif 'adversarial' in method:
            if method == 'adversarial':
                func = **train_adversarial**
            elif method == 'train_adversarial_multiteacher':
                func = **train_adversarial_multiteacher**
            elif method == 'train_adversarial_multiteacher_wgangp':
                func = **train_adversarial_multiteacher_wgangp**
